### PR TITLE
PXBLUE-2609 Removed px blue references

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes #  PXBLUE-2609

#### Changes proposed in this Pull Request:
- pxblue ==> brightlayer-ui

### CODE REVIEW

File: mui/CHANGELOG.md [hunk 1]
```diff
@@ -113,5 +113,5 @@ Previous versions listed after this indicator refer to our deprecated `@pxblue`
 
 ```tsx
 // new import syntax
-import IconName from '@pxblue/icons-mui/IconName';
+import IconName from '@brightlayer-ui/icons-mui/IconName';
```
Comment (emclaug2) [inline mui/CHANGELOG.md:116]: I wouldn't change any pxb refs in the older changelogs; these notes probably don't even apply to this package anymore.

**Comment by emclaug2** [comment]
How is this building for you?

```
C:/Users/Evan/Documents/git/pxblue/icons/mui/demos/showcase/src/App.tsx
TypeScript error in C:/Users/Evan/Documents/git/pxblue/icons/mui/demos/showcase/src/App.tsx(2,24):
Could not find a declaration file for module '@brightlayer-ui/icons-mui'. 'C:/Users/Evan/Documents/git/pxblue/icons/mui/demos/showcase/node_modules/@brightlayer-ui/icons-mui/index.js' implicitly has an 'any' ty
pe.
  Try `npm i --save-dev @types/brightlayer-ui__icons-mui` if it exists or add a new declaration (.d.ts) file containing `declare module '@brightlayer-ui/icons-mui';`  TS7016

    1 | import React from 'react';
  > 2 | import * as Icons from '@brightlayer-ui/icons-mui';
      |                        ^
    3 | import { Grid } from '@material-ui/core';
    4 | const meta = require('@brightlayer-ui/icons-mui/index.json');
    5 |

```

I just did a fresh clone of the repo; ran `cd mui && yarn start`

**Review by daileytj — APPROVED** [review]
Looks good to me

**Comment by daileytj** [comment]
Screen Shot 2021-11-23 at 4 20 40 PM

**Review by jeffvg — APPROVED** [review]
looks good

**Comment by daileytj** [comment]
@emclaug2 do you want to give this one another go before merging?

**Pull request merged**